<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <include file="$(find navsatfix2pose)/launch/setup_gnssposetf.launch">
  </include>

  <arg name="sub_gnss_topic_name" default="/f9p/fix"/>
  <arg name="pub_rtk_topic_name" default="/rtk_base_link"/>
  <arg name="pub_rtk_covariance_topic_name" default="/sensing/gnss/pose_with_covariance"/>
  <arg name="pub_rtk_fix_topic_name" default="/fixed/pose"/>
  <param name="sub_gnss_topic_name" value="$(arg sub_gnss_topic_name)"/>
  <param name="pub_rtk_topic_name" value="$(arg pub_rtk_topic_name)"/>
  <param name="pub_rtk_covariance_topic_name" value="$(arg pub_rtk_covariance_topic_name)"/>
  <param name="pub_rtk_fix_topic_name" value="$(arg pub_rtk_fix_topic_name)"/>

  <param name="plane" value="7"/>
  <param name="tf_num" value="1"/>
  <!-- 1 : plane  2 : mgrs  -->
  <param name="convert_height_num" value="0"/>
  <!-- 0 : No convert  1 : ellipsoid -> altitude  2 : altitude -> ellipsoid -->
  <node pkg="navsatfix2pose" name="navsatfix2pose" type="navsatfix2pose" output="screen"/>
</launch>
